playbook/workshops/remote_control/README.md

```diff
@@ -0,0 +1,615 @@
+
+##### Referencing the button from JavaScript
+
+Now that we set the button's id, we add this to our JavaScript:
+
+```js
+document.getElementById("motivation-button").onclick = function() {
+    [PUT ANY CODE HERE YOU WANT WHEN THE BUTTON IS CLICKED]
+};
+```
+_Don't include the `[PUT ANY CODE HERE YOU WANT WHEN THE BUTTON IS CLICKED]`_
+
+> ![](img/getElement.gif)
+
+There is a lot to understand in the above code but if you read it out loud like
+this, it might begin to make sense:
+
+_Note that I highlighted the keywords from the code_
+
+In the `document`, `get` the `element` with the `id` `"motivation-button"` and
+when it is `clicked`, run any of the code that is inside the `function`.
+
+There are definitely more concepts and complexities here but we won't worry
+about them in this workshop. If you want to read more about what's happening
+here, feel free to Google `javascript function` and `document.getElementById`
+and/or ask a facilitator!
+
+##### Adding an alert
+
+Now we can add the code inside of the `function`.
+
+Let's add what's called an `alert` to the JavaScript
+
+`alert("You pressed the button");`
+
+> ![](img/alert.gif)
+
+Now when you click the button, it should create a pop-up, but JS Bin blocks
+them in this view. You'll need to click the black arrow to pop the output out:
+
+> ![](img/popopen.gif)
+
+Now click the `Motivate Partner`
+
+> ![](img/motivate_partner_alert.gif)
+
+> If it doesn't, ask a neighboring group to see if they can see what's wrong.
+> Otherwise, raise your hand to ask a facilitator for help!
+
+##### Adding the actual code to make the phone call:
+
+Let's add the phone calling functionality.
+
+Don't forget to change the `555-555-5555` phone number to your own.
+
+```js
+document.getElementById("dead-people-btn").onclick = function() {
+  Twilio.callAndSay("555-555-5555", "Time to wake up! Keep hacking!");
+  alert("You pressed the button.");
+};
+```
+
+> ![](img/wake_up.gif)
+
+If you try pressing the `Motivate Partner` button, you should get a phone call!
+
+> If it doesn't, ask a neighboring group to see if they can see what's wrong.
+> Otherwise, raise your hand to ask a facilitator for help!
+
+
+#### Trying it out on mobile
+
+This website is responsive so let's open it on your phone.
+
+Just click "Share" and then "Output only" to get the URL
+
+> ![](img/responsive.gif)
+
+Then type in the URL into your mobile browser. (You can also be clever and
+write a line of code to send the URL to your phone with a text message).
+
+### The final code
+
+Here's what [my code looks like]
+(http://jsbin.com/fawuda/120/edit?html,js,console,output)
+right now. In order for this example to work:
+
+- change the phone number to your phone number
+- set your twilio api key and token using
+  [the instructions here](#adding-your-twilio-sid-and-token)
+
+## Part II
+
+### Making the examples work
+
+**IMPORTANT:**
+
+All the examples also need to have a Twilio sid and auth token. However
```

Comment:
`auth`→`authentication` unless it is specifically called an authentication token.